src/components/ContentUploader/ItemAction.js

```diff
@@ -6,15 +6,18 @@
 import React from 'react';
 import { injectIntl } from 'react-intl';
 import PlainButton from 'box-react-ui/lib/components/plain-button/PlainButton';
+import Tooltip from 'box-react-ui/lib/components/tooltip';
```

Comment:
Tooltips use tether. Ideally we want to put the tethered tooltip in the scope of the box-element DOM, and not in the body. You may have to eventually do something similar to what I did for drop down menu in this commit:

https://github.com/box/box-ui-elements/commit/225d56938cc45c6d613eab11af1c8c6467777e08#diff-89ee97fcc3d518c05eac1416189d85c8R26

Also see corresponding commit in box-react-ui, to take bodyElement and pass it to tether.

If the tooltip is tethered to the <body> which happens by default, then it may not get all the CSS/reset styles, since the <body> tag has no class and is out of scope from the box-element's perspective.

Reply:
I am going to spend some time next week to see if migrating from tether.js to popper.js is dooable and easy.
https://medium.com/@FezVrasta/popper-js-v1-5e8b3acd888c

Reply:
Got it, I'll look into that.
But I think for now it's fine to just use tether, since the Box webapp will be the only user to the UploadsManager for a while, and the tooltip will be styled properly in the context of the webapp. We can revisit this once we decide to put up more documentation and make it easy for 3rd party developers to use as well.